shims/spark300/src/main/scala/com/nvidia/spark/rapids/shims/spark300/Spark300Shims.scala

```diff
@@ -158,8 +160,15 @@ class Spark300Shims extends SparkShims {
           override def convertToGpu(): GpuExec = {
             val sparkSession = wrapped.relation.sparkSession
             val options = wrapped.relation.options
+
+            val location = getLocationWithAlluxioReplace(
+              conf,
+              wrapped.relation,
+              wrapped.partitionFilters,
+              wrapped.dataFilters)
```

Comment:
what happens if the user runs a query with input_file_name()? does it show original or alluxio:// paths? I think it uses alluxio one but want to verify and we need to decide if that is what we want. its probably something we just document.

Something like:

      ```
   lambda spark : spark.read.parquet(data_path)\
                    .filter(f.col('a') > 0)\
                    .selectExpr('a',
                        'input_file_name()',
                        'input_file_block_start()',
                        'input_file_block_length()'),
```

Reply:
it shows alluxio:// paths, I just verified

Reply:
we need to document this.  can you please file an issue to add documentation for the entire feature and include this

Reply:
filed [an issue ](https://github.com/NVIDIA/spark-rapids/issues/1574) to track the document things